pkcs11helpers/helpers.go

```diff
@@ -182,26 +182,28 @@ func Sign(ctx PKCtx, session pkcs11.SessionHandle, object pkcs11.ObjectHandle, k
 	return signature, nil
 }
 
+var ErrNoObject = errors.New("no objects found matching provided template")
+
 // FindObject looks up a PKCS#11 object handle based on the provided template.
 // In the case where zero or more than one objects are found to match the
 // template an error is returned.
 func FindObject(ctx PKCtx, session pkcs11.SessionHandle, tmpl []*pkcs11.Attribute) (pkcs11.ObjectHandle, error) {
 	if err := ctx.FindObjectsInit(session, tmpl); err != nil {
 		return 0, err
 	}
-	handles, more, err := ctx.FindObjects(session, 1)
+	handles, _, err := ctx.FindObjects(session, 10)
```

Comment:
Any particular reason to choose 10 instead of, say, 2? Seems like a magic number whose value isn't 0, 1, or -1, so should probably be explained with a comment.

Reply:
You're right, 2 is correct here.